### PR TITLE
Update iammeter.markdown

### PR DESCRIPTION
**Pull request by lewei50**
Add the description of how to use IAMMETER products to monitor the solar pv system in Home Assistant

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/iammeter.markdown [hunk 1]
```diff
@@ -13,12 +13,14 @@ ha_platforms:
   - sensor
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
+[Monitor your solar PV system with IAMMETER energy monitor in Home Assistant](https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710)
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]: I don't think we should link to community forum resources from our documentation.

File: source/_integrations/iammeter.markdown [hunk 2]
```diff
@@ -13,12 +13,14 @@ ha_platforms:
...
 
 Example Lovelace Dashboard:
 
 <p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
+  ![IAMMETER energy monitor in Home Assistant](https://iammeterglobal.oss-ap-southeast-1.aliyuncs.com/img/image-20211108143148254.png)
+  ![monitor the solar pv system in Home Assistant](https://iammeterglobal.oss-ap-southeast-1.aliyuncs.com/img/image-20211108143245427.png)
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]: images need to be hosted from our repository, not external. This is for privacy reasons.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/iammeter.markdown (hunk 2 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: Already done. Thanks

File: source/_integrations/iammeter.markdown (hunk 1 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: Hi Frenck:
So glad to receive your message again.
I have removed this content temporarily.
But in fact, we want to introduce some specific solutions that use our products in HA here.
Such as how to monitor the solar PV system? how to use the Grafana to display data while using HA to get the data?
Has it permissible to publish such content in the doc?

If so , could we use the content post in our website or the HA official forum?

Thanks for your attention.
BR
Jason

Comment (frenck) [reply source/_integrations/iammeter.markdown]: > Such as how to monitor the solar PV system? how to use the Grafana to display data while using HA to get the data?
> Has it permissible to publish such content in the doc?

We are not the Grafana docs. Things like these can be made available on the community forums or your own resources but are not what we want to have in our end-user documentation.

File: source/_integrations/iammeter.markdown [hunk 3]
```diff
@@ -13,12 +13,13 @@ ha_platforms:
...
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
 
 Example Lovelace Dashboard:
 
 <p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
+<img src='/images/integrations/iammeter/energy-monitor-Home Assistant.png' />
+<img src='/images/integrations/iammeter/solar-pv-monitoring-home-assistant.png' />
 </p>
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]: - Please use markdown to include images
- The first filename has spaces, please make it formatted like the second filename
- The first screenshot just shows the Home Assistant energy dashboard, how is that related to this integration?
- The second screenshot is a custom configuration, this is misleading as the end-user might think they will end up getting that when setting up this integration, which is not true.

File: source/_integrations/iammeter.markdown [hunk 4]
```diff
@@ -13,12 +13,13 @@ ha_platforms:
   - sensor
 ---
 
-`iammeter` provides real-time readings of single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com) over Wi-Fi.
+`IAMMETER` provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]:
```suggestion
IAMMETER provides real-time readings of single-phase (WEM3080) and three-phase (WEM3080T) meters from [IAMMETER](https://www.iammeter.com/docs/homeassistant) over Wi-Fi.
```

Comment (frenck) [reply source/_integrations/iammeter.markdown]: Why does this link to docs and not a product?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/iammeter.markdown (hunk 1 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: OK

File: source/_integrations/iammeter.markdown (hunk 3 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: Please use markdown to include images (**got it**)
The first filename has spaces, please make it formatted like the second filename (**got it**)

The first screenshot just shows the Home Assistant energy dashboard, how is that related to this integration?
The second screenshot is a custom configuration, this is misleading as the end-user might think they will end up getting that when setting up this integration, which is not true.

(**In fact, these two pics are related to this  forum post, https://community.home-assistant.io/t/monitor-your-solar-pv-system-in-home-assistant-energy-dashboard-vs-energy-management/342710
I added this in the first commit and removed it according to your suggestion in the second commit.
Without the introduction of the post, these two pics are indeed easy to mislead the user.
Could I add a section at the last of this doc called "How to monitor your energy consumption" and put these two pics and the forum link there?**)

File: source/_integrations/iammeter.markdown (hunk 4 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: I think this link is like a step-by-step tutorial about how to integrate the IAMMETER products into HA.

Comment (frenck) [reply source/_integrations/iammeter.markdown]: Yes, but the link doesn't suggest that.

File: source/_integrations/iammeter.markdown (hunk 3 above)
Comment (frenck) [reply source/_integrations/iammeter.markdown]: If you want to guide users on how to set up energy monitoring with Home Assistant, we should link to that specific page in Home Assistant.

The energy dashboard screenshot is not a feature of this integration, it is a Home Assistant feature. It doesn't belong here.

Comment (lewei50) [reply source/_integrations/iammeter.markdown]: understood,I have already remove such pics in this doc

File: source/_integrations/iammeter.markdown (hunk 4 above)
Comment (lewei50) [reply source/_integrations/iammeter.markdown]: Hi, Frenck, 
Thanks for your suggestion.
I have replaced the tutorial page with the official web page of IAMMETER and added two products page.

**Comment by lewei50** [comment]
Hi Frenck 
looking forward to your further suggestion.
Thanks

**Comment by lewei50** [comment]
Hi:
So appreciate your attention.
If possible, please let me know whether there are any other questions in this PR.

**Comment by frenck** [comment]
Stop calling out messages, please...! Be patient instead.

File: source/_integrations/iammeter.markdown [hunk 5]
```diff
@@ -11,17 +11,17 @@ ha_codeowners:
...
-<p class='img'>
-  <img src='/images/integrations/iammeter/demo.jpg' />
-</p>
+[IAMMETER](https://www.iammeter.com/) provides both bi-directional single-phase energy meters ([WEM3080](https://www.iammeter.com/products/single-phase-meter)) and bidirectional three-phase energy meters  ([WEM3080T](https://www.iammeter.com/products/three-phase-meter))  that can be integrated into Home Assistant.
+
+
+
 
-## Configuration
+# Configuration
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]: This change is incorrect

File: source/_integrations/iammeter.markdown [hunk 6]
```diff
@@ -49,44 +49,44 @@ name:
   default: IamMeter
 {% endconfiguration %}
 
-## Sensors
+# Sensors
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]: This change is incorrect, please revert.

**Review by frenck — CHANGES_REQUESTED** [review]
There are now unused images attached in this PR, please clean that up.

**Comment by lewei50** [comment]
> Stop calling out messages, please...! Be patient instead.

Sorry, I am too anxious before.
Already done.
Please check it at your convenience.
Thanks.

File: source/_integrations/iammeter.markdown [hunk 7]
```diff
@@ -11,15 +11,14 @@ ha_codeowners:
   - '@lewei50'
 ha_platforms:
   - sensor
+
+
```
Comment (frenck) [inline source/_integrations/iammeter.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]

**Pull request merged**